Assets/SEE/Net/Network.cs

```diff
@@ -288,33 +294,106 @@ private void Start()
             Instance = this;
 
             NetworkManager.Singleton.OnServerStarted += OnServerStarted;
+            NetworkManager.Singleton.NetworkConfig.ConnectionApproval = true;
+            ProcessCommandLineArguments();
         }
 
         /// <summary>
-        /// Initializes the server, client and game.
+        /// Processes and determines the values of the command-line arguments
+        /// <see cref="ServerPort"/>, <see cref="RoomPassword"/>, <see cref="BackendDomain"/>,
+        /// <see cref="ServerId"/>, and starts the server if the command-line argument
+        /// <see cref="launchAsServerArgument"/> is present.
         /// </summary>
-        private void StartUp()
+        /// <exception cref="ArgumentException">thrown if an option requiring a value does
+        /// not have one</exception>
+        private void ProcessCommandLineArguments()
         {
-#if UNITY_EDITOR
-            if (networkCommsLoggingEnabled)
+            string[] arguments = Environment.GetCommandLineArgs();
+
+            // Check command line arguments
+            // The first element in the array contains the file name of the executing program.
+            // If the file name is not available, the first element is equal to String.Empty.
+            for (int i = 1; i < arguments.Length; i++)
             {
-                NetworkComms.EnableLogging(new NetworkCommsLogger(minimalSeverity));
+                if (arguments[i] == portArgument)
+                {
+                    CheckArgumentValue(arguments, i, portArgument);
+                    ServerPort = Int32.Parse(arguments[i + 1]);
+                    i++;
+                }
+                else if (arguments[i] == passwordArgument)
+                {
+                    CheckArgumentValue(arguments, i, passwordArgument);
+                    RoomPassword = arguments[i + 1];
+                    i++;
+                }
+                else if (arguments[i] == domainArgument)
+                {
+                    CheckArgumentValue(arguments, i, domainArgument);
+                    BackendDomain = arguments[i + 1];
+                    i++;
+                }
+                else if (arguments[i] == serverIdArgument)
+                {
+                    CheckArgumentValue(arguments, i, serverIdArgument);
+                    ServerId = arguments[i + 1];
+                    i++;
+                }
+                else if (arguments[i] == launchAsServerArgument)
+                {
+                    StartServer(null);
+                }
```

Comment:
This forces that argument to come last, right? If so, maybe we should set a boolean instead and execute the server after the for loop in order not to miss any subsequent arguments.